ktor-server/ktor-server-core/jvm/src/io/ktor/features/OriginConnectionPoint.kt

```diff
@@ -20,14 +20,13 @@ public val ApplicationRequest.origin: RequestConnectionPoint
 /**
  * A key to install a mutable [RequestConnectionPoint]
  */
-@KtorExperimentalAPI
+@Deprecated("This is going to be deprecated once the mutableConnectionPoint becomes public.")
```

Comment:
Should we postpone deprecation once there is a replacement? Now users can't do anything about it and will just suppress deprecation.